great_expectations/render/renderer/slack_renderer.py

```diff
@@ -116,10 +116,6 @@ def render(
                 }
                 query["blocks"].append(dataset_element)
 
-        custom_blocks = self._custom_blocks(evr=validation_result)
-        if custom_blocks:
-            query["blocks"].append(custom_blocks)
-
```

Comment:
This code may have been an artifact from previous behavior. `_custom_blocks()` function only returns `None`, means this block would never have run.